### DISCUSSION TITLE
Which emission factors are used from EPA

### DISCUSSION THREAD

**Discussion by andiosika** [USEPA/supply-chain-factors#74] [Q&A] [answered]
Trying to pinpoint which EPA GHG factors you are using? Which Version? https://cfpub.epa.gov/si/si_public_record_Report.cfm?dirEntryId=349324&Lab=CESER thanks

**Comment by bl-young** [comment, marked as answer]
This code currently produces v1.2 of the factors. You can see the series of releases here: https://github.com/USEPA/supply-chain-factors/releases which aligns with prior versions of the factors.

**Reply by andiosika** [reply]
are these the factors used in the USEEIO v2.0.1-411? Trying to kee it all straight.

**Reply by bl-young** [reply]
Each model is denoted by a specific model spec. USEEIO v2.0.1-411 was published a few years back. You can view the data vintages in that [model spec](https://github.com/USEPA/useeior/blob/master/inst/extdata/modelspecs/USEEIOv2.0.1-411.yml). Once a model is published, it typically will not change. Note that that model includes about 15 different satellite tables (GHGs, and many others).

The supply chain factors are a product that can be constructed from any model using the [code in this repository](https://github.com/USEPA/supply-chain-factors/blob/main/CalculateEmissionFactors.Rmd#L27). EPA has only published supply chain factors for GHGs, though one could build a set of supply chain factors for any of the satellite tables if a model like USEEIO v2.0.1-411 was used as the basis. The most recent set of factors are based on this model spec: [USEEIOv2.1.19-GHG](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.1.19-GHG.yml). The GHG data in this model are more current (2019) than the GHG data used in USEEIO v2.0.1-411, otherwise the underlying model construction is the same (specifically you can compare the parameters in the first 12 rows or so of each model spec to see they are identical):
```
BaseIOSchema: 2012
BaseIOLevel: "Detail"
IOYear: 2012 # Year for IO data
ModelRegionAcronyms: ["US"]
ModelType: "EEIO"
IODataSource: "BEA"
BasePriceType: "PRO" #producer
BasewithRedefinitions: FALSE
CommodityorIndustryType: "Commodity"
ScrapIncluded: FALSE
DisaggregationSpecs: "WasteDisaggregationDetail"
```
